src/lib/undux/utils/account.js

```diff
@@ -23,7 +23,7 @@ const updateAll = store => {
       }
     })
     .catch(e => {
-      log.error('updateAll failed', e.message, e)
+      log.warn('updateAll failed', e.message, e)
```

Comment:
catch balanceof and checkentitlement exceptions before "then", this catch should handle other errors like from "store" and it should still log.error them

Reply:
done

Reply:
@serdiukov-o-nordwhale its not what i requested

Reply:
@sirpy you mean to do like the following:
https://github.com/GoodDollar/GoodDAPP/pull/1596/commits/f9c4efe35a30e5edcee7cc24cdf0eeb31426c854#diff-abe5f06beb36f9f32e9eb3bb52f0d840R10

?

Reply:
yes